### ISSUE TITLE
Keeps using the same background

### ISSUE THREAD

**Issue by johnwc** [onlineth/Spotlight-Desktop#8]
It doesn't seem to be working anymore. It just keeps using the same background every time, and it seems to be blurry. Nothing like the sharp Spotlight photos that are usually used. I am a developer, is there something I can do to debug?

Running on Win 10 1703

**Comment by onlineth** [comment]
Hi John,

Ya, it would help a lot if you could take a look at the following locations in the registry,

1. HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Authentication\LogonUI\Creative\[An ID]\[Sub node (whichever is the last one)]\
1. HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Authentication\LogonUI\Creative\[An ID] - (No subnodes)
1. SOFTWARE\Microsoft\Windows\CurrentVersion\Lock Screen\Creative

Basically, it would help to know where it's picking up that blurry image. Your build seems to be pretty close to mine, so this was my setup where that image you see is my current lock screen image,

![](https://i.imgur.com/0qeydZc.jpg)

**Comment by johnwc** [comment]
It seems to be an issue with Windows I believe. There is a LockScreenOption that is set to 0 in HKEY_LOCAL_MACHINE\SOFTWARE\Microsoft\Windows\CurrentVersion\Authentication\LogonUI\Creative\{SID}. If I manualy set it to 4 and reapply the settings in the Lock Screen settings page it works. But it seems that this keeps getting set back to 0 after some time.

**Comment by onlineth** [comment]
Ya, Windows on the backend when it retrieves photos updates all that itself. Can you check [this guide](https://github.com/onlineth/Spotlight-Desktop/wiki/Spotlight-Known-Locations-in-Registry) and see if the image you get is also blurry?

**Comment by MagedSaeed** [comment]
It seems that the same problem is happening with me as well. The image is the same I guess, It is not blurry. However, I do not know what to do with the registry values there. Any help Mr @onlineth

**Comment by onlineth** [comment]
Hmm, can you check display settings?

Right click on the desktop,
![image](https://user-images.githubusercontent.com/7810121/36281586-c8b7fe08-1263-11e8-9ec4-fde42fc70b98.png)

And make sure choose fit is set to stretch,
![image](https://user-images.githubusercontent.com/7810121/36281611-dbeb3cba-1263-11e8-9f0d-1dcc393ab684.png)

**Comment by johnwc** [comment]
I updated to 1709 and that registry entry stopped effecting the images coming in, so it now works as expected.

**Comment by MagedSaeed** [comment]
I have uninstalled it and now I am downloading it again to check.
Hope it will work perfectly this time.
👍

**Comment by onlineth** [comment]
Awesome, I'm going to close this but reopen if you experience this problem again.
